pkg/expr/sql/frame_db_conv.go

```diff
@@ -40,11 +39,22 @@ func convertToDataFrame(ctx *mysql.Context, iter mysql.RowIter, schema mysql.Sch
 		}
 
 		for i, val := range row {
-			v, err := fieldValFromRowVal(f.Fields[i].Type(), val)
+			// Run val through mysql.Type.Convert to normalize underlying value
+			// of the interface
+			nV, _, err := schema[i].Type.Convert(val)
+			if err != nil {
+				return nil, err
+			}
+
+			// Run the normalized value through fieldValeToRow to normalize
+			// the interface type to the dataframe value type, and make nullable
+			// values pointers as dataframe expects.
+			fV, err := fieldValFromRowVal(f.Fields[i].Type(), nV)
```

Comment:
Can I ask what `fV` and `nV` stand for here 🙏 ? No big deal of course, but just curious. I guess `fV` is "field value", but not sure about `nV`. Thanks!